Sming/compiler/ld/eagle.app.v6.cpp.include.ld

```diff
@@ -0,0 +1,217 @@
+/*  
```

Comment:
Dear @slaff, please put the PR on hold. Further test showed that there might be some impacts that I do not fully understand. I don't know why, but the new _rboot.rom0.ld_ seems not to be completely equivalent the old one. In one of my projects I get garbled sming debug output, possible caused by a wrong linker setting. I already did some research, but have not found the reason yet. Better safe than sorry ;-)